.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5.1.0
         with:
-          go-version: '1.22.2'
+          go-version: '1.23.2'
```

Comment:
Any reason to not use `1.23.5` ?
Also, does this action need to install a specific version? What if it just installed latest?

Reply:
@adrianmoisey, to make the go version f.or ci pipeline for GitHub workflow use the same version as we used for our projects